src/lando/api/tests/mocks.py

```diff
@@ -1819,3 +1815,86 @@ def _build_handlers(self):
             if hasattr(getattr(self, a), "_conduit_method")
         ]
         return {handler._conduit_method: handler for handler in handlers}
+
+
+class TreeStatusDouble:
```

Comment:
This change looks like it was accidentally removed in the merge PR and re-added here. It is already in `main`.